### ISSUE TITLE
allow easy access to memberships in a term

### ISSUE THREAD

**Issue by mhl** [everypolitician/everypolitician-popolo-python#9]
At the moment, to find all the memberships associated with a term, you have to do something like:

```python
for membership in popolo.memberships:
    if membership.legislative_period_id != latest_term.id:
        continue
    # do whatever you like with the membership...
```

It would be nicer to be able to do:

```python
for membership in latest_term.memberships:
    # do whatever you like with the membership
```

**Comment by andylolz** [comment]
+1! The ruby version has this feature, it’s really useful.

It would also be great to have a simpler method for getting the *current* membership… I think this can be found by filtering memberships from the current legislative period with `end_date=ApproxDate.FUTURE`. Is that right?

I suspect this is a common enough usecase that it should perhaps appear in the README.

EDIT: in the ruby version, `current_*` is deprecated in favour of `latest_*` which makes sense.

**Comment by andylolz** [comment]
I thiiink this one was fixed in ead67a20.

**Comment by mhl** [comment]
Yep, good point - I'm closing this
